lib/puppet/resource.rb

```diff
@@ -480,7 +480,7 @@ def lookup_in_environment(name, scope)
   def lookup_in_module(name, scope)
     found = false
     value = catch(:no_such_key) do
-      v = Puppet::DataProviders.lookup_in_module(name, Puppet::DataBinding::LookupInvocation.new(scope), nil)
+      v = Puppet::DataProviders.lookup_in_module(name, Puppet::Pops::Lookup::Invocation.new(scope), nil)
       found = true
```

Comment:
Puppet::Resource is also agent side, but i suppose the methods lookup_xxx never gets called there - raises the issue of pops initialization on agent side otherwise... (not sure what we concluded there).